### DISCUSSION TITLE
Dynamic event list needs ATCB

### DISCUSSION THREAD

**Discussion by xziler8** [add2cal/add-to-calendar-button#468] [Q&A]
I have a PHP page that builds a dynamic event list -- each item I want to add an ATCB (custom icon that triggers script)
I see the example configs the script at the page end -- I conversely may have 12 events. Without writing a static config script for each I'm  at a loss.
Would like to be able to call the function triggering the script from each event (calendar icon) with parameters passed.
Please assist.

**Comment by jekuer** [comment]
Hi @xziler8 ,

That's simply a JavaScript development task.
The atcb_action function is very straight forward. If you want to optimize the code on your end of the table, I would recommend to use ChatGPT or hire a JS developer via Upwork.

All the best!
Jens
